acceptance_tests/mixins.py

```diff
@@ -0,0 +1,17 @@
+from urlparse import urljoin
+
+from django.test import TestCase
+from selenium import webdriver
+
+
+class BaseTestCase(TestCase):
+    HOST = 'http://127.0.0.1:8080/'
+
+    def setUp(self):
+        self.driver = webdriver.Chrome()
+        self.login_url = urljoin(self.HOST, 'users/login')
+        self.register_url = urljoin(self.HOST, 'users/register')
+        self.user_profile_url = urljoin(self.HOST, 'users/user-profile')
```

Comment:
Not all test cases care for these URLs. Define each of them in the class that uses them at the class level, like the HOST is defined within this class.